config/purposes/scrna_core_cell_extraction.yml

```diff
@@ -86,34 +86,52 @@ LRC PBMC Bank:
       units: 'cells/ml'
       default_threshold: 650000
       decimal_places: 0
-# FluidX tube for freezing and output of cell banking protocol
-LRC Bank Seq:
-  :asset_type: tube
-  :target: SampleTube
-  :type: Tube::Purpose
+# Tube rack for storing LRC Bank Seq tubes
+TR LRC Bank Seq:
+  :asset_type: tube_rack
+  :target: TubeRack
+  :size: 96
+  :type: TubeRack::Purpose
   :creator_class:
     name: LabwareCreators::PlateSplitToTubeRacks
     args: &fluidx_tube_creation_config
       child_seq_tube_purpose_name: LRC Bank Seq
       child_seq_tube_name_prefix: SEQ
+      child_seq_tube_rack_purpose_name: TR LRC Bank Seq
       child_spare_tube_purpose_name: LRC Bank Spare
       child_spare_tube_name_prefix: SPR
-  :ancestor_stock_tube_purpose_name: LRC Blood Vac
-  :presenter_class: Presenters::SimpleTubePresenter
-  :state_changer_class: StateChangers::AutomaticTubeStateChanger
+      child_spare_tube_rack_purpose_name: TR LRC Bank Spare
+      child_tube_rack_metadata_key: 'tube_rack_barcode'
```

Comment:
We want this key to be the same EVERYWHERE, right? It's part of Abdullah's design doc?
So it doesn't feel quite right here against a specific purpose.

Reply:
How do I reference something like a TubeRack class constant from within a yml file?

Reply:
if it will ALWAYS be the same key then there is no need for Limber to know what it is or include it in the purpose config. instead of passing it through for each tube rack in the attributes, I can pull it out of a tuberack constants initializer file like I made in the other story on the SS side.